content/blog/2022-05-17-aws-remotes-in-dvc.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Using AWS Remotes in DVC
+date: 2022-05-17
+description: >
+  We're going to set up an AWS S3 remote in a DVC project.
+descriptionLong: >
+  It can be tricky setting up a remote to handle data versioning in DVC so we're
+  going to go through a tutorial for doing this with AWS.
+picture: 2022-05-17/aws-in-dvc.png
+pictureComment: Using AWS Remotes in DVC
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/preventing-stale-models-in-production/1137
+tags:
+  - MLOps
+  - DVC
+  - Git
+  - DVC Remotes
+  - Collaboration
+---
+
+When you’re working on a data science project that has huge datasets, it’s
+common to store them in cloud storage. You’ll also be working with different
+versions of the same datasets to train a model, so it’s crucial to have a tool
+that enables you to do this quickly and easily. That’s why we’re going to do a
+quick walkthrough of how to set up a remote in an AWS S3 bucket and handle data
+versioning with [DVC](https://dvc.org/doc).
```

Comment:
💅🏼 Not sure what's your practice in blogs but as-is it will show an "external" link (special icon, opens in new window), I think. Or do this for a regular (internal) link:

```suggestion
versioning with [DVC](/doc).
```

There's a few other instances of https://dvc.org/... links